.github/workflows/ci.yaml

```diff
@@ -14,31 +14,31 @@ on:
   workflow_call:
 
 jobs:
-  lint:
-    name: Lint
-    runs-on: ubuntu-latest
-    timeout-minutes: 5
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Install tox
-        # TODO: Consider replacing with custom image on self-hosted runner OR pinning version
-        run: python3 -m pip install tox
-      - name: Run linters
-        run: tox run -e lint
-
-  unit-test:
-    name: Unit tests
-    runs-on: ubuntu-latest
-    timeout-minutes: 15 # TODO: reduce to 5 minutes
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Install tox
-        # TODO: Consider replacing with custom image on self-hosted runner OR pinning version
-        run: python3 -m pip install tox
-      - name: Run tests
-        run: tox run -e unit
```

Comment:
This PR should be rebased on `main` that is ONLY using Juju3.

The ugly commented out stuff should "natively" disappear after the rebase (conflicts resolved, sry :sweat_smile: )